tests/zerocashTest.cpp

```diff
@@ -382,8 +533,11 @@ bool SimpleTxTest(const size_t tree_depth) {
     cout << "\nSIMPLE TRANSACTION TEST\n" << endl;
 
     libzerocash::timer_start("Param Generation");
-    libzerocash::ZerocashParams p(tree_depth);
-    p.getProvingKey(1); // Strangely enough, this is how we trigger parameter generation.
+    auto keypair = libzerocash::ZerocashParams::GenerateNewKeyPair(tree_depth);
+    libzerocash::ZerocashParams p(
+        tree_depth,
+        &keypair
+    );
```

Comment:
[does not block] We're generating parameters in multiple tests.  Can we generate them once and reuse them for all tests that aren't testing generation itself? Wouldn't this be a big time save? Worth a followup ticket.